.github/workflows/test_and_publish.yml

```
@@ -84,10 +84,13 @@ jobs:
         psql -h localhost -p 5432 -U gis -d gis -c 'CREATE SCHEMA gis;'
 
         # Add PostGIS extension to "gis" database
-        psql -h localhost -p 5432 -U gis -d gis -c 'CREATE EXTENSION IF NOT EXISTS postgis;'
+        psql -h localhost -p 5432 -U gis -d gis -c 'CREATE EXTENSION IF NOT EXISTS postgis SCHEMA public;'
 
-        # Drop PostGIS Tiger Geocoder extension to "gis" database
-        psql -h localhost -p 5432 -U gis -d gis -c 'DROP EXTENSION IF EXISTS postgis_tiger_geocoder CASCADE;'
+        # Add PostGISRaster extension to "gis" database
```

```suggestion
        # Drop PostGIS Tiger Geocoder extension to "gis" database
        psql -h localhost -p 5432 -U gis -d gis -c 'DROP EXTENSION IF EXISTS postgis_tiger_geocoder CASCADE;'

        # Add PostGISRaster extension to "gis" database
```
Seems it's actually still needed...

---

okay